docs/howto/craft-a-snap/example-cross-compiled-app.rst

```diff
@@ -0,0 +1,161 @@
+.. _example-cross-compiled-app:
+
+Example Cross-Compiled App
+==========================
+
+Projects built using the :ref:`craft_parts_autotools_plugin` can be cross-compiled,
+allowing software to be developed on one architecture and built for a number of target
+architectures.
+
+Overview
+--------
+
+There are a number of steps to enable cross-compilation for an autotools project:
+
+#. Specify the architectures used.
+#. Adjust the options passed to the ``configure`` script.
+#. Include packages to support cross-compilation.
+#. Handle linker warnings.
+
+Example project file for example-curl
+-------------------------------------
+
+The following code comprises the project file for a project that uses ``libcurl``. The
+complete code can be found in `this repository <https://github.com/mr-cal/hello-curl>`_.
+
+.. collapse:: hello-curl project
+
+  .. literalinclude:: ../code/craft-a-snap/example-cross-compiled-app-recipe.yaml
+      :caption: snapcraft.yaml
+      :language: yaml
+
+The ``platforms`` key
+-------------------------
```

Comment:
I think this heading would be better if it spoke to a task and contained an imperative verb. How about "Target an architecture"?